src/components/RequestsFilters/RequestsFilters.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Accordion,
+  FilterAccordionHeader,
+} from '@folio/stripes/components';
+import {
+  CheckboxFilter,
+  MultiSelectionFilter,
+} from '@folio/stripes/smart-components';
+
+import {
+  requestFilterTypes,
+  requestStatusFilters,
+  requestTypeFilters,
+} from '../../constants';
+
+export default class RequestsFilters extends React.Component {
+  static propTypes = {
+    activeFilters: PropTypes.objectOf(PropTypes.array).isRequired,
+    resources: PropTypes.object.isRequired,
+    onChange: PropTypes.func.isRequired,
+    onClear: PropTypes.func.isRequired,
+  };
+
+  transformRequestFilterOptions = (source = []) => {
+    return source.map(({ label, value }) => ({
+      label: <FormattedMessage id={label} />,
+      value,
+    }));
+  };
+
+  transformTagsOptions = () => {
+    return get(this.props.resources, 'tags.records', []).map(({ label }) => ({
+      label,
+      value: label,
+    }));
+  };
+
+  render() {
+    const {
+      activeFilters: {
+        tags = [],
+        requestType = [],
+        requestStatus = [],
+      },
+      onChange,
+      onClear,
+    } = this.props;
+
+    return (
+      <React.Fragment>
+        <Accordion
+          displayClearButton={requestType.length > 0}
```

Comment:
Could you please use isEmpty for this for clarity?
```suggestion
          displayClearButton={isEmpty(requestType)}
```